src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPayment = days * 40;
```

Comment:
Move all the magic numbers 40, 50, 20 to constants

Reply:
Done)

Reply:
What is 40? Magic number

Reply:
Sorry, my bad, forget to commit last change of 40 value